tests/test_integrations.py

```diff
@@ -0,0 +1,74 @@
+from uuid import uuid1
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def integrations_create_webhook_incoming(logged_rocket):
+    return logged_rocket.integrations_create(
+        integrations_type="webhook-incoming",
+        name=str(uuid1()),
+        enabled=True,
+        username=logged_rocket.me().json().get("username"),
+        channel="#general",
+        script_enabled=False,
+    ).json()
+
+
+def test_integrations_create(integrations_create_webhook_incoming, logged_rocket):
+    integration_webhook_incoming = integrations_create_webhook_incoming
+    assert integration_webhook_incoming.get("success")
+
+    integration_webhook_outgoing = logged_rocket.integrations_create(
+        integrations_type="webhook-incoming",
+        name=str(uuid1()),
+        enabled=True,
+        event="sendMessage",
+        username=logged_rocket.me().json().get("username"),
+        urls=["http://example.com/fake"],
```

Comment:
```suggestion
        urls=["https://example.com/fake"],
```
Just to make SonarCloud happy